nucleus/annotation.py

```diff
@@ -46,18 +47,17 @@ class Annotation:
     @classmethod
     def from_json(cls, payload: dict):
         """Instantiates annotation object from schematized JSON dict payload."""
-        if payload.get(TYPE_KEY, None) == BOX_TYPE:
-            return BoxAnnotation.from_json(payload)
-        elif payload.get(TYPE_KEY, None) == POLYGON_TYPE:
-            return PolygonAnnotation.from_json(payload)
-        elif payload.get(TYPE_KEY, None) == CUBOID_TYPE:
-            return CuboidAnnotation.from_json(payload)
-        elif payload.get(TYPE_KEY, None) == CATEGORY_TYPE:
-            return CategoryAnnotation.from_json(payload)
-        elif payload.get(TYPE_KEY, None) == MULTICATEGORY_TYPE:
-            return MultiCategoryAnnotation.from_json(payload)
-        else:
-            return SegmentationAnnotation.from_json(payload)
+        type_key_to_type: Dict[str, Type[Annotation]] = {
```

Comment:
Needed to move this to a dict lookup because `pylint` will block if you have more than 6 `return`s in one function.